test/test_es_utils.py

```diff
@@ -26,4 +27,34 @@ def test_lucene_query_basic(es_client_fixture):
         'sort': [{'_uid': {'order': 'desc'}}]
     }
     results = execute_lucene_query_on_es(client=es_client_fixture, index='fourfront-mastertestuser', query=test_query)
-    assert len(results) == 1
\ No newline at end of file
+    assert len(results) == 1
+
+
+@pytest.mark.integrated
+def test_get_bulk_uuids_embedded(es_client_fixture):
+    """ Tests getting some bulk uuids acquired from search. """
+    uuids = ['1a12362f-4eb6-4a9c-8173-776667226988']  # only one uuid first
+    result = get_bulk_uuids_embedded(es_client_fixture, 'fourfront-mastertestuser', uuids)
+    assert len(result) == 1
```

Comment:
For each of these, I would perhaps minimally also do something to test that the set of results has the indicated uuids in it. But also and perhaps more important, if there is a way to test that it's getting the embedded view and not some other view, that would be good, perhaps by checking that a certain key was included that is part of the embedded view and that some other is excluded that is part of a more expansive view.